app/presenters/tree_builder_vms_and_templates.rb

```diff
@@ -51,12 +51,14 @@ def reparent_hidden_folders(tree)
   end
 
   def prune_rbac(tree)
-    vms = extract_vms(tree).uniq
-
-    allowed_vm_ids = Set.new(Rbac.filtered(vms).map(&:id)) # results_format == :ids
+    # results_format == :ids
+    allowed_vm_ids = if User.current_user.settings[:display_vms]
```

Comment:
@kbrock in a meeting about this, I thought we had agreed with @serenamarie125, @jonnyfiveiq, @dclarizio, @Fryguy and @dmetzger57 that this would be per-user setting, NOT appliance setting.  Meeting attendees - please correct me if I am wrong.